setup.py

```diff
@@ -1,21 +1,54 @@
+import codecs
+import os
+import re
+
 from setuptools import setup, find_packages
 
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+
+def read(*parts):  # Stolen from txacme
+    with codecs.open(os.path.join(HERE, *parts), 'rb', 'utf-8') as f:
+        return f.read()
+
+
+def readme():
+    # Prefer the ReStructuredText README, but fallback to Markdown if it hasn't
+    # been generated
+    if os.path.exists('README.rst'):
+        return read('README.rst')
+    else:
+        return read('README.md')
```

Comment:
I did this in marathon-acme because the README is markdown and I used pandoc to convert it to rst. So you probably don't need this function.